index.html

```diff
@@ -163,13 +170,39 @@ <h2>
       </dd>
       <dt>
         <dfn data-dfn-for="digital credential" data-local-lt=
-        "response">Presentation response</dfn>
+        "presentation response">Presentation response</dfn>
       </dt>
       <dd>
         A format that a [=holder|holder's=] software, such as a digital wallet,
         uses, via an [=digital credential/exchange protocol=], to respond to a
         [=digital credential/request data=] by a [=verifier=].
       </dd>
+      <dt>
+        <dfn data-dfn-for="digital credential" data-local-lt=
+        "issuance request">Issuance request</dfn>
```

Comment:
No need to duplicate the linked term. 

```suggestion
        <dfn data-dfn-for="digital credential" data-local-lt="issuance">Issuance request</dfn>
```

Reply:
Done!